client/build-project.sh

```diff
@@ -69,9 +68,8 @@ IP=$(ip -f inet addr show eth1 2> /dev/null | grep 'inet' | awk '{ print $2}' |
 #[ $? -eq 0 ] && echo -e "Build ${BUILD} started.\nYou can watch builds progress at https://${IP}:8443/console/project/${NAME}/browse/builds"
 
 echo "==> Send build configs to build tube"
-python $CWD/send_build_request.py ${NAME} ${TAG} ${DEPENDS_ON}
+python $CWD/send_build_request.py ${NAME} ${TAG} ${REPO_BRANCH} ${REPO_BUILD_PATH} ${TARGET_FILE} ${NOTIFY_EMAIL} ${DEPENDS_ON}
```

Comment:
@navidshaikh Are we planning for change in tube job flow? as these details are only used for build_worker which does not need any of these for now?